src/App.js

```diff
@@ -14,11 +16,91 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+const goods = goodsFromServer.map(good => ({
+  name: good, isSelected: false,
+}));
+
+class App extends React.Component {
+  state = {
+    selectedGoods: [...goods],
```

Comment:
```suggestion
    selectedGoods: [...goods],
```
selected значит уже выбранные, у тебя изначально все выбранные?